src/main/kotlin/lotto/model/winning/WinningCounter.kt

```diff
@@ -0,0 +1,47 @@
+package lotto.model.winning
+
+import lotto.model.LottoTickets
+import lotto.model.Money
+import lotto.model.winning.WinningPlace.FIRST
+import lotto.model.winning.WinningPlace.SECOND
+import lotto.model.winning.WinningPlace.THIRD
+import lotto.model.winning.WinningPlace.FOURTH
+import lotto.model.winning.WinningPlace.FIFTH
+import lotto.model.winning.WinningPlace.MISS
+
+data class WinningCounter private constructor(
+    private val counter: Map<WinningPlace, Int>
+) : Map<WinningPlace, Int> by counter {
+    val priceSum: Money
+        get() {
+            return counter.toMutableMap().entries.fold(Money.ZERO, sumWinningsTimesCount())
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            return entries.fold(Money.ZERO, sumWinningsTimesCount())
```